src/text-input-autocomplete-menu.component.ts

```diff
@@ -53,6 +61,27 @@ export class TextInputAutocompleteMenuComponent {
   get choices() {
     return this._choices;
   }
+  @HostBinding('style.top.px')
+  get top() {
+    const screenHeight = window.screen.height;
```

Comment:
This will also break compatibility with universal, you can get the window from the injected document though like `this.document.defaultView`